src/controllers/Lane.js

```diff
@@ -200,7 +200,7 @@ class Lane extends Component {
     const { components } = this.props
     const pickedProps = pick(
       this.props,
-      ['id','label','title','titleStyle','cards', 'labelStyle','t','inlineEditTitle','canAddLanes']
+      ['id','label','title','titleStyle','cards', 'labelStyle','t','inlineEditTitle','canAddLanes', 'draggable', 'laneDraggable']
     )
     return (
       <components.LaneHeader {...pickedProps} onDelete={this.removeLane} onDoubleClick={this.toggleLaneCollapsed} updateTitle={this.updateTitle}/>
```

Comment:
we can pass one prop only:

```laneDraggable={draggable && laneDraggable}```

BTW I think to remove `draggable` property from Lane-s controller at all in near future.